session-1/oscar-arrieta/contar.rb

```diff
@@ -0,0 +1,11 @@
+def contar(array = [])
+   cont = 5.to_i
+  if array.include?(cont)
+    "#{ array.count(5) }"
+  else
+   "0"
+  end
+end
+puts contar([1, 5, 5, 1]) # 2
```

Comment:
don't forget use the correct identation.

```suggestion
def contar(array = [])
  array.count(5)
end

puts contar([1, 5, 5, 1]) # 2
```

Reply:
done